### DISCUSSION TITLE
Welcome to forum Discussions!

### DISCUSSION THREAD

**Discussion by atmoio** [standardnotes/forum#2458] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bubbaxen** [comment]
Noob Here. I can't seem to get the web clipper working in Brave. Works fine in Firefox. Am I missing something?

Thanks,
Trygve
